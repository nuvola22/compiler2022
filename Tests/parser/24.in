begin
    writeln(1);
    begin
        writeln(2);
        begin
            writeln(3);
            begin
                writeln(4);
            end;
            writeln(5);
        end;
        writeln(6);
        begin
            writeln(7);
        end;
        writeln(8);
    end;
    writeln(9);
    writeln(10);
end.