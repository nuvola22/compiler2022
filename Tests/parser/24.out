      program
         begin
            call
               writeln
               1
            begin
               call
                  writeln
                  2
               begin
                  call
                     writeln
                     3
                  begin
                     call
                        writeln
                        4
                  call
                     writeln
                     5
               call
                  writeln
                  6
               begin
                  call
                     writeln
                     7
               call
                  writeln
                  8
            call
               writeln
               9
            call
               writeln
               10
